octo/utils/gym_wrappers.py

```diff
@@ -274,19 +274,24 @@ def unnormalize(self, data, metadata):
         if self.normalization_type == "normal":
             return (data * metadata["std"]) + metadata["mean"]
```

Comment:
Can we make all of this 1-1 with octo.data.utils.data_utils

https://github.com/octo-models/octo/blob/5bd39e8cf7ae24545a071b84f95242fe4176a492/octo/data/utils/data_utils.py#L219